src/CMakeLists.txt

```diff
@@ -602,6 +602,8 @@ install(
     PATTERN "*.h"
     PATTERN "*.pac"
     PATTERN "3rdparty/*" EXCLUDE
+    # Headers used only during build
+    PATTERN "threading/formatters/detail" EXCLUDE
```

Comment:
Do we even need to exclude it? The header is in a detail filesystem namespace and this one-off seems weird to me.

Reply:
I agree that it's annoying for a one-off in the CMake file, but that it shouldn't be installed if external consumers aren't going to use it.